[2025-09-06T04:24:48.824+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: mysql_insert_select.validate_model manual__2025-09-06T04:24:27.007585+00:00 [queued]>
[2025-09-06T04:24:48.841+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: mysql_insert_select.validate_model manual__2025-09-06T04:24:27.007585+00:00 [queued]>
[2025-09-06T04:24:48.841+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 1
[2025-09-06T04:24:48.863+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): validate_model> on 2025-09-06 04:24:27.007585+00:00
[2025-09-06T04:24:48.876+0000] {standard_task_runner.py:57} INFO - Started process 134 to run task
[2025-09-06T04:24:48.882+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'mysql_insert_select', 'validate_model', 'manual__2025-09-06T04:24:27.007585+00:00', '--job-id', '188', '--raw', '--subdir', 'DAGS_FOLDER/mysql_insert_select.py', '--cfg-path', '/tmp/tmpv9nbvjde']
[2025-09-06T04:24:48.885+0000] {standard_task_runner.py:85} INFO - Job 188: Subtask validate_model
[2025-09-06T04:24:48.961+0000] {task_command.py:410} INFO - Running <TaskInstance: mysql_insert_select.validate_model manual__2025-09-06T04:24:27.007585+00:00 [running]> on host 6faa580ba6cb
[2025-09-06T04:24:49.087+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='mysql_insert_select' AIRFLOW_CTX_TASK_ID='validate_model' AIRFLOW_CTX_EXECUTION_DATE='2025-09-06T04:24:27.007585+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-06T04:24:27.007585+00:00'
[2025-09-06T04:24:49.089+0000] {logging_mixin.py:149} INFO - ✅ Modelo cargado correctamente
[2025-09-06T04:24:49.089+0000] {logging_mixin.py:149} INFO -    - Tipo: <class 'numpy.ndarray'>
[2025-09-06T04:24:49.089+0000] {logging_mixin.py:149} INFO -    - Características esperadas: No disponible
[2025-09-06T04:24:49.089+0000] {python.py:183} INFO - Done. Returned value was: None
[2025-09-06T04:24:49.109+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=mysql_insert_select, task_id=validate_model, execution_date=20250906T042427, start_date=20250906T042448, end_date=20250906T042449
[2025-09-06T04:24:49.174+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2025-09-06T04:24:49.219+0000] {taskinstance.py:2674} INFO - 1 downstream tasks scheduled from follow-on schedule check
